docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -68,9 +68,12 @@ refs:
 
 # Table
 
-Tables are a highly flexible visualization designed to display data in columns and rows. They support various data types, including tables, time series, annotations, and raw JSON data. The table visualization can even take multiple data sets and provide the option to switch between them. With this versatility, it's the preferred visualization for viewing multiple data types, aiding in your data analysis needs.
+Tables are a highly flexible visualization designed to display data in columns and rows.
+They support various data types, including tables, time series, annotations, and raw JSON data.
```

Comment:
Also this is a bit odd - what is a table(s) data type? or time series? 🤔

Reply:
So this is what it said back in v10.3 before our last round of content updates. Does this make more sense?

> Tables are very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data.

Reply:
I'm not convinced 😅 @leeoniya, any ideas?

Reply:
I mean time-series data is a kind of data. I'll fight you on that, lol. But I can take out "tables". I feel like by saying "column and rows" we kind of cover that.

Reply:
yeah, I think that's fine. I don't have super strong opinions, just the wording around data types feels a bit odd. I think it's ok either way.

Reply:
Actually I think since that sentence isn't clear, we should just take it out. What do you think?
```suggestion
```

Reply:
i'm not going to attempt wordsmithing here, but my general observation is that Table can render [as rows and columns] any and all data that our core panels, datasource plugins, and app plugins support ([DataFrames](https://grafana.com/developers/plugin-tools/key-concepts/data-frames)).

i dont know if there's value in trying to itemize a subset of DataFrame shapes when the real answer is that they can render "everything".

Reply:
Thanks for weighing in @leeoniya. I'm just gonna take that sentence out.